projects/repast/src/types.js

```diff
@@ -0,0 +1,10 @@
+import { trim, pipe, contains, split } from "ramda";
+
+// @repast isCompositeType :: String -> Boolean
+export const isCompositeType = pipe(trim, ([h, ...t]) => h === "{");
```

Comment:
Minor, but you might wanna do:
```js
import {trim, pipe, contains, split, head, equals} from 'ramda'

export const isCompositeType = pipe(trim, head, equals('{'))
```

Also, somehow the linting here is not working as I expect, but we should move these single characters into the `CHARACTER` / `C` constants in `src/constants.js`

So likely: `leftBrace: '{'` + `pipe: '|'`?

Reply:
Also, the destruction ([h, ...t]), which same as getting first character of a string?

```
"abcd"[0]
'a'
```

coud be

```js
> R.head("abcd")
'a'
```

Reply:
Good point yes, I like the head version more I'd say.